docs/_data/table/cluster-management.csv

```diff
@@ -1,16 +1,16 @@
 propertyName,defaultValue
 alluxio.integration.master.resource.cpu,1
 alluxio.integration.master.resource.mem,1024 MB
-alluxio.integration.mesos.executor.dependency.path,https://s3.amazonaws.com/alluxio-mesos
-alluxio.integration.mesos.jre.path,jre1.7.0_79
-alluxio.integration.mesos.jre.url,https://s3.amazonaws.com/alluxio-mesos/jre-7u79-linux-x64.tar.gz
+alluxio.integration.mesos.executor.dependency.path,http://downloads.alluxio.org/downloads/files/${alluxio.version}/alluxio-${alluxio.version}-bin.tar.gz
+alluxio.integration.mesos.jdk.path,jdk1.7.0_79
+alluxio.integration.mesos.jdk.url,https://s3-us-west-2.amazonaws.com/alluxio-mesos/jdk-7u79-linux-x64.tar.gz
```

Comment:
would something like `https://alluxio-mesos.s3.amazonaws.com/jdk-7u79-linux-x64.tar.gz` work? I'm wondering if could get rid of the `us-west-2` part.